packages/math-input/src/index.ts

```diff
@@ -41,5 +41,8 @@ export type {default as Keys} from "./data/keys";
 export {default as KeyConfigs} from "./data/key-configs";
 export {type KeyType, KeypadType} from "./enums";
 
+// Keypad Configuration type
+export {type KeypadConfiguration} from "./types";
```

Comment:
We already have `export type {KeypadAPI} from "./types";`, could we add `KeypadConfiguration` to that?